### PR TITLE
Feature/25478 vaccine scheduling design updates

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
update vaccine schedule flow with design changes from review

## Testing done
local and unit test

## Screenshots
**/appointments/new-covid-19-vaccine-booking**

image

<hr />

**/appointments/new-covid-19-vaccine-booking/contact-facilities**

image

## Acceptance criteria
- [ ] matches [design mockup](https://www.sketch.com/s/fd383f03-0827-49a7-865c-e0b44f70b183/a/52PWQyP#Inspector) for /appointments/new-covid-19-vaccine-booking/contact-facilities
- [ ] matches design [mockup](https://www.sketch.com/s/fd383f03-0827-49a7-865c-e0b44f70b183/a/3OPEQLD#Inspector) for /appointments/new-covid-19-vaccine-booking

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/vaos/covid-19-vaccine/components/ContactFacilitiesPage.jsx [hunk 1]
```diff
@@ -73,14 +73,14 @@ export default function ContactFacilitiesPage() {
...
         <>
+          <h1>{pageTitle}</h1>
           <AlertBox
             className="vads-u-margin-top--0"
-            level="1"
+            level="2"
             status="warning"
             backgroundOnly
-            headline={pageTitle}
+            headline="If you got your first dose:"
```
Comment (jbalboni) [inline src/applications/vaos/covid-19-vaccine/components/ContactFacilitiesPage.jsx:82]: In the design, it doesn't look like this should be a header/headline, but might want to check with Lauren or Peter, if you haven't.

File: src/applications/vaos/redux/selectors.js [hunk 2]
```diff
@@ -66,3 +66,6 @@ export const selectFeatureVAOSServiceVAAppointments = state =>
 
 export const selectFeatureVAOSServiceCCAppointments = state =>
   toggleValues(state).vaOnlineSchedulingVAOSServiceCCAppointments;
+
+export const selectFeatureVariantTesting = state =>
+  toggleValues(state).vaOnlineSchedulingVariantTesting;
```
Comment (jbalboni) [inline src/applications/vaos/redux/selectors.js:71]: I think this is from another PR?

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 3]
```diff
@@ -81,6 +81,7 @@ export default Object.freeze({
   vaOnlineSchedulingVAOSServiceCCAppointments: 'vaOnlineSchedulingVAOSServiceCCAppointments',
   vaOnlineSchedulingVAOSServiceRequests: 'vaOnlineSchedulingVAOSServiceRequests',
   vaOnlineSchedulingVAOSServiceVAAppointments: 'vaOnlineSchedulingVAOSServiceVAAppointments',
+  vaOnlineSchedulingVariantTesting: 'vaOnlineSchedulingVariantTesting',
```
Comment (jbalboni) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:84]: Same comment as above.

File: src/applications/vaos/covid-19-vaccine/components/ContactFacilitiesPage.jsx (hunk 1 above)
Comment (jenniemc) [reply src/applications/vaos/covid-19-vaccine/components/ContactFacilitiesPage.jsx:82]: Just checked with Peter. Said he prefers having a header in the alert box.

File: src/applications/vaos/redux/selectors.js (hunk 2 above)
Comment (jenniemc) [reply src/applications/vaos/redux/selectors.js]: Oh good catch. I should have closed all my file tabs in VS Code before doing a checkout.

File: src/platform/utilities/feature-toggles/featureFlagNames.js (hunk 3 above)
Comment (jenniemc) [reply src/platform/utilities/feature-toggles/featureFlagNames.js]: thanks for catching

**Review by alexandec — APPROVED** [review]

**Review by jbalboni — APPROVED** [review]

**Pull request merged**